### ISSUE TITLE
mount: unknown filesystem type 'vboxsf' on Windows

### ISSUE THREAD

**Issue by pareshchouhan** [dr4g0nnn/VagrantGazelle#7]
On windows it might fail due to no vbox addons being installed with the error 
```
Vagrant was unable to mount VirtualBox shared folders. This is usually
because the filesystem "vboxsf" is not available. This filesystem is
made available via the VirtualBox Guest Additions and kernel module.
Please verify that these guest additions are properly installed in the
guest. This is not a bug in Vagrant and is usually caused by a faulty
Vagrant box. For context, the command attempted was:

mount -t vboxsf -o uid=1000,gid=1000 var_www_ /var/www

The error output from the command was:

mount: unknown filesystem type 'vboxsf'
````
It can be fixed by installing vagrant-vbguest

```
vagrant plugin install vagrant-vbguest
vagrant destroy && vagrant up
```  

Kindly add it to README.md, or I can do a PR if you want, above commands and this config worked for me.

```
config.vm.synced_folder "src/", "/var/www/", nfs: true
```

**Comment by dr4g0nnn** [comment]
Hi @pareshchouhan,

Given that I can't test this (I don't have any Windows machines), could you do a pull request for it?

Thanks!

**Comment by dr4g0nnn** [comment]
Fixed by #8

**Comment by vitorhub** [comment]
first rule witch works for me is that i closed chrome, players, any other program, installed guestadditions as pareshchouhan said before and everything works fine  for me.
